parcels/tools/_helpers.py

```diff
@@ -56,3 +62,37 @@ def deprecated_made_private(func: Callable) -> Callable:
 
 def patch_docstring(obj: Callable, extra: str) -> None:
     obj.__doc__ = f"{obj.__doc__ or ''}{extra}".strip()
+
+
+def pretty_field(field: Field) -> str:
+    """Return a pretty repr for Field"""
+    out = f"""<{type(field).__name__}>
+    grid            : {field.grid!r                    }
+    extrapolate time: {field.allow_time_extrapolation!r}
+    time_periodic   : {field.time_periodic!r           }
+    gridindexingtype: {field.gridindexingtype!r        }
+    to_write        : {field.to_write!r                }
+"""
+    return dedent(out).strip()
+
+
+def pretty_particleset(pset: ParticleSet) -> str:
+    """Return a pretty repr for ParticleSet"""
+    if len(pset) < 10:
+        particles = repr(list(pset))
+    else:
+        lst = [repr(pset[i]) for i in range(7)] + ["..."]
+        particles = f"[{', '.join(lst)}]"
+
+    out = f"""<{type(pset).__name__}>
+    fieldset   : {pset.fieldset}
+    pclass     : {pset.pclass}
+    repeatdt   : {pset.repeatdt}
+    # particles: {len(pset)}
+    particles  : {particles}
```

Comment:
All the particles are now written as one long line. Can we print each particle position on a separate line?